### ISSUE TITLE
[deps] "parcel" not found - pls fix or document

### ISSUE THREAD

**Issue by DoubleU23** [SoftwareBrothers/better-docs#45]
had to install "parcel" manually

**FYI:** im currently on a windows machine - if that is an issue

**Comment by wojtek-krysiak** [comment]
Hi - thanks for using it. Unfortunately parcel cannot be a dependency because better docs can be used without it (if you don't have @component tags in the code). In the docs, I've written it under "Installation instructions" in @component plugin section.

Nevertheless probably not everyone reads the docs, so what I would suggest is to maybe add an error explaining that parcel has to be installed globally.

**Comment by DoubleU23** [comment]
Thank you  ery much for the explanation. Wasn't sure about that.

**Comment by sbmadhav** [comment]
@wojtek-krysiak - does parcel have to be a global dependecy? Can't we add it as a devDependency in our code? Appreciate any leads here.

**Comment by wojtek-krysiak** [comment]
when we put it as devDependency - it means that all users will have to install it even if they don't use @component plugin. That is why it has to be installed as a global dependency.

**Comment by myreli** [comment]
Old thread but an update: 

Since 5.2.x npm provides [npx](https://github.com/npm/npx) out of the box, which allows us to use `parcel` as a development dep without the need for a global install. As an enhancement, a command-line interface could let the user choose between adding or not `parcel-bundler` as a peer dependency during better-docs install. The same tool could generate the most common setups. 

Now with a single `npm install jsdoc better-docs parcel-bundler --save-dev` everything works as expected. Maybe that could be added to the docs.

**Comment by wojtek-krysiak** [comment]
yep CLI will be perfect but this is rather bigger task - don't have this in the TODO right now :(
